### PR TITLE
{WIP}increased timeout to create exception record{do not merge with master}

### PR DESCRIPTION
**Pull request by vakepati**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java [hunk 1]
```diff
@@ -84,7 +84,7 @@ public void should_attach_exception_record_to_the_existing_case_with_no_evidence
 
         //then
         await("Exception record is attached to the case")
-            .atMost(60, TimeUnit.SECONDS)
+            .atMost(1200, TimeUnit.SECONDS)
```
Comment (rkondratowicz) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java]: 20 minutes??

**Comment by rkondratowicz** [comment]
Please remember to update the PR description.
The prefilled template text says what needs to be done.

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java (hunk 1 above)
Comment (vakepati) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java]: sorry Just for debugging purpose i increased timeout,i just change the title of the PR with WIP

**Comment by lgonczar** [comment]
If we know it didn't help, is it ok to close this PR?

**Comment by vakepati** [comment]
This PR is no more valid

**Pull request closed without merging**